applications/GeoMechanicsApplication/custom_constitutive/tension_cutoff.h

```diff
@@ -28,7 +28,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TensionCutoff : public YieldSurface
 
     explicit TensionCutoff(double TensileStrength);
 
-    [[nodiscard]] double YieldFunctionValue(const Vector& rPrincipalStress) const override;
+    [[nodiscard]] double YieldFunctionValue(const Vector& rSigmaTau) const override;
     [[nodiscard]] Vector DerivativeOfFlowFunction(const Vector& rPrincipalStress) const override;
```

Comment:
And "unname" it here too:
```suggestion
    [[nodiscard]] Vector DerivativeOfFlowFunction(const Vector&) const override;
```